clients/native/src/client/config/template.rs

```diff
@@ -110,10 +110,15 @@ host = '{{ socket.host }}'
 
 [debug]
 
+[[debug.traffic]]
```

Comment:
Is this a backwards incompatible change that will require re-init?

Reply:
old configs should now get updated during `run`